app/src/main/java/ingsw/group1/findmyphone/activity/ModifyContactActivity.java

```diff
@@ -0,0 +1,81 @@
+package ingsw.group1.findmyphone.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.eis.smslibrary.SMSPeer;
+
+import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.contacts.SMSContactSwipeCallback;
+import ingsw.group1.findmyphone.contacts.SMSContact;
+import ingsw.group1.findmyphone.contacts.SMSContactManager;
+
+/**
+ * Activity for adding a new contact
+ *
```

Comment:
Careful here.
Also I'd add info about the fact that only the name can be modified